tests/process/spawn.luau

```diff
@@ -114,27 +150,31 @@ end
 
 task.cancel(thread2)
 
-local sleepElapsed = os.clock() - sleepStart
+local sleepElapsed = (os.clock() - sleepStart) - win_commandExecutionTolarance
+
 assert(
 	sleepElapsed >= SLEEP_DURATION,
 	"Spawning a process that does blocking sleep did not sleep enough"
 )
 assert(
-	sleepElapsed < SLEEP_DURATION * 1.5,
+	sleepElapsed < yieldTolarance,
 	"Coroutine yielded the main lua thread during process yield"
 )
 
 -- Inheriting stdio & environment variables should work
 
 local echoMessage = "Hello from child process!"
 local echoResult = process.spawn("echo", {
-	'"$TEST_VAR"',
+	if isWindows then '"$Env:TEST_VAR"' else '"$TEST_VAR"',
 }, {
 	env = { TEST_VAR = echoMessage },
-	shell = "bash",
+	shell = if isWindows then true else "bash", -- bash does not exist on Windows, using default shell option instead.
```

Comment:
Since we explicitly passed `"bash"` here I think it's fine to also explicitly pass `"powershell"` when on Windows instead of `true`